README.md

```diff
@@ -100,6 +109,7 @@ pre-commit run --all-files
 Each time you start a new terminal instance you will need to run the following commands to get the server running again:
 
     source env/bin/activate
+    pip install -r requirements/generated/requirements-dev.txt
```

Comment:
You don't need to run this every time you start a new terminal, it should only need to be run once